src/components/v5/common/CompletedAction/partials/PaymentBuilder/partials/PaymentBuilderWidget/hooks.tsx

```diff
@@ -145,7 +145,9 @@ export const useGetFinalizeStep = ({
             )}
 
             {selectedFinalizeAction && !selectedFinalizeMotion && (
-              <ActionWithPermissionsInfo action={selectedFinalizeAction} />
+              <FinalizeByPaymentCreatorInfo
+                userAdddress={selectedFinalizeAction.initiatorAddress}
+              />
             )}
```

Comment:
Here we should perform a similar check for `userIsCreator` to decide which description to actually show